db/00000000000001_initial_schema.up.sql

```diff
@@ -207,7 +207,8 @@ CREATE TABLE report_aggregations(
     client_timestamp    TIMESTAMP NOT NULL,                 -- the client timestamp this report aggregation is associated with
     ord                 BIGINT NOT NULL,                    -- a value used to specify the ordering of client reports in the aggregation job
     state               REPORT_AGGREGATION_STATE NOT NULL,  -- the current state of this report aggregation
-    prep_transition     BYTEA,                              -- the current preparation transition (opaque VDAF message, only if in state WAITING)
+    prep_state          BYTEA,                              -- the current preparation transition (opaque VDAF message, only if in state WAITING)
```

Comment:
This column can be a VDAF transition (for the Leader) or a VDAF preparation state (for the Helper); therefore, IMO the column name is confusing, since it would imply to me that the column contains a preparation state.

We could either:
(a) introduce two nullable columns, `prep_transition` & `prep_state`, and fill them out as appropriate for the type of the task.
(b) name this column something generic enough that containing either a transition or a prep state makes sense (`prep_data`? `vdaf_data`?), and update the comment to describe what the contents might be.

I'd prefer (a) -- that's pretty much what we've done elsewhere -- but (b) would work too. Mostly I just don't want the column name to imply that this is a preparation state.

Reply:
I went with (a) and named the columns `helper_prep_state` and `leader_prep_transition` to further disambiguate their purpose.